apps/storybook/.storybook/itwinTheme.js

```diff
@@ -62,16 +62,16 @@ export const darkTheme = create({
     '"Noto Sans Mono", ui-monospace, "SF Mono", SFMono-Regular, "DejaVu Sans Mono", Menlo, Consolas, monospace',
 
   // Text colors
-  textColor: 'rgba(255,255,255,0.85)',
+  textColor: '#dbdbdb',
 
   // Toolbar default and active colors
-  barTextColor: 'rgba(255,255,255,0.85)',
-  barSelectedColor: '#A5D7F5',
-  barBg: '#4F5D65',
+  barTextColor: '#dbdbdb',
+  barSelectedColor: '#008ee0',
```

Comment:
I think some of these colors in dark theme are wrong. It should not be this dark:
![](https://user-images.githubusercontent.com/9084735/200884054-ca2971fa-dbd1-477e-8ab0-de734e59e75f.png)

Also can you put comments next to all these hex values mentioning which variables we are using?